jplag/src/main/java/de/jplag/CommandLineArgument.java

```diff
@@ -136,4 +184,56 @@ private String retrieveDescriptionFromMessages() {
         }
         return Messages.getString(getClass().getSimpleName() + "." + builder.toString());
     }
+
+    private static class Builder {
```

Comment:
I've made this Builder Nested because it makes no sense to use it outside of the scope of this class.
And since it is private I felt that a generic name suffices.

Should I still move it outside and rename it?

Reply:
I would say leave it in for now.